src/main.rs

```diff
@@ -457,4 +482,4 @@ fn main() {
     stream.play().unwrap();
 
     app.run();
-}
\ No newline at end of file
+}
```

Comment:
My editor believes in newlines at the end of files, and I just gave up fighting it. Sorry!

Reply:
Newlines for the newline gods. It's okay :)